src/com/lesson/TutByPage2.java

```diff
@@ -0,0 +1,59 @@
+package com.lesson;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class TutByPage2 {
+
+    @FindBy(css = "a[data-target-popup='authorize-form']")
+    private WebElement enterPopup;
+
+    private WebElement login;
+
+    private WebElement password;
+
+    @FindBy(css = "input.button.auth__enter")
+    private WebElement enterButton;
+
+    @FindBy(className = "uname")
+    private WebElement content;
+
+    private WebDriver driver;
```

Comment:
I recommend to make WebDriver instance final in page objects, so you will never change it